src/api/app/views/layouts/webui/responsive_ux/_bottom_navigation.html.haml

```diff
@@ -8,7 +8,7 @@
           .small Watchlist
       %li.nav-item.border-left.border-gray-500
         - tasks = User.session!.tasks
```

Comment:
@krauselukas, regarding the tasks button. Right now, the [top](https://github.com/openSUSE/open-build-service/pull/9248/files#diff-876c1049a6b090974a80d9b52150266cL13) and [bottom](https://github.com/openSUSE/open-build-service/pull/9248/files#diff-876c1049a6b090974a80d9b52150266cL13) navigation are different

Reply:
upps, will check that

Reply:
done